staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -334,12 +334,12 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 			return nil
 		}
 		if err != nil {
-			if !apierrors.IsInvalid(err) {
-				return err
-			}
-			klog.Warning("The watch-list feature is not supported by the server, falling back to the previous LIST/WATCH semantics")
+			// originally we honored only apierrors.IsInvalid
```

Comment:
History doesn't really matter [especially that it wasn't really in prod anywhere] - let's remove this part.

Let's switch the comment to sth like:

We fallback to list on every error because it is ...

Reply:
pushed, PTAL.